src/test/java/it/gov/pagopa/payment/BaseIntegrationTest.java

```diff
@@ -17,14 +17,19 @@
 import org.awaitility.core.ConditionTimeoutException;
 import org.junit.jupiter.api.BeforeAll;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.mongo.MongoProperties;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.kafka.test.context.EmbeddedKafka;
 import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.web.servlet.MockMvc;
 
 @SpringBootTest
+@EmbeddedKafka(topics = {
+        "spring.cloud.stream.bindings.notificationOutcome-out-0.destination"
```

Comment:
Keep attention when you use properties name! without ${..} notation, you are just using the property name as value itself!
```suggestion
        "${spring.cloud.stream.bindings.notificationOutcome-out-0.destination}"
```